app/views/application/references/index.html

```diff
@@ -15,7 +15,7 @@
     <div class="govuk-grid-column-two-thirds">
       <h1 class="govuk-heading-l">{{ title }}</h1>
 
-      <p class="govuk-body">You need to give details of 2 people who can give a reference for you. They’ll be only be contacted if you accept an offer.</p>
+      <p class="govuk-body">You need to give the email address of 2 people who can give a reference for you. They’ll be only be contacted if you accept an offer.</p>
```

Comment:
When [copying this change to the production codebase](https://github.com/DFE-Digital/apply-for-teacher-training/pull/7952), I realised there’s a typo here ("be only be" 🤦) and also on production it says "an offer on a course" rather than "an offer" – not sure which of those is better?

```suggestion
      <p class="govuk-body">You need to give the email address of 2 people who can give a reference for you. They’ll only be contacted if you accept an offer on a course..</p>
```

Reply:
I think 'on a course' is better! Good spot!